bin/setup

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env ruby
+require 'pathname'
+
+# path to your application root.
+APP_ROOT = Pathname.new File.expand_path('../../',  __FILE__)
+
+# This can't be a rake task because the app won't spinup without these files.
+def copy_sample_files
+  [
+    "config/secrets.yml",
+    "config/database.yml"
+  ].each do |config|
+    sample = config + ".sample"
+    unless File.exist?(config)
+      system "cp #{sample} #{config}"
+    end
+  end
+end
+
+Dir.chdir APP_ROOT do
+  puts "== Installing dependencies =="
+  system "gem install bundler --conservative"
+  system "bundle check || bundle install"
+
+  puts "\n== Copying sample files =="
+  copy_sample_files
+
+  puts "\n== Randomizing secrets =="
+  system "bin/rake setup:randomize_secrets"
+
+  puts "\n== Preparing database =="
+  if ENV['RAILS_ENV'] == "test"
+    system "bin/rake db:test:prepare"
```

Comment:
FYI: This is deprecated in the version of rails that this app uses. Is there a better way to do this now?